.gitlab-ci.yml

```diff
@@ -18,7 +18,7 @@ before_script:
 
 # ================== templates ================== #
 .base_template: &base_template
-  image: 486234852809.dkr.ecr.us-east-1.amazonaws.com/ci/corp-ci:master
+  image: 486234852809.dkr.ecr.us-east-1.amazonaws.com/ci/corp-ci:zach_bump-hugo
```

Comment:
```suggestion
  image: 486234852809.dkr.ecr.us-east-1.amazonaws.com/ci/corp-ci:mastter
```

Reply:
Hi Kaylyn, so for this, can we first merge this PR, then merge the corp-ci PR https://github.com/DataDog/corp-ci/pull/61 in case anything happens, we don't then have to revert corp-ci, we can just change the image name back.

Reply:
one this is merged, and corp-ci, we can change the image name back to `master`